start

```diff
@@ -448,17 +450,17 @@ function run_silent() {
   local ARGS=$@
   local OUTFILE="/tmp/run_silent.out"
 
-  echo -n "$LABEL: "
```

Comment:
Why don't we print the name of the command that failed? I think this change would make it harder to determine what had failed. Or will the command be outputted in some other way?

Reply:
fixed that, prints label and status on same line was net intended change.